updatecli/updatecli.d/manifest.yaml

```diff
@@ -160,7 +160,8 @@ actions:
     kind: github/pullrequest
     scmid: turtles
     spec:
-      automerge: false
+      automerge: true
```

Comment:
if I am understanding correctly, we should avoid auto-merge on updatecli PRs since there could be a situation where it might bump to the version of provider compatible with x version of CAPI, but we still use x-1 version of CAPI in turtles or vice versa.

Reply:
We have a number of required approvals, this will not be merged automatically. If such change gets into repo - it is after series of reviews and approves.